### ISSUE TITLE
Status Message

### ISSUE THREAD

**Issue by BaneOfSerenity** [lostisland/faraday#590]
While using Faraday I noticed that the status code is in the response.status but there is no way to get the status message. When looking at my VCR cassette there is a code and a message being sent back. It would be nice to be able to access this message especially for 4xx and 5xx returns. Also it seem the message is being thrown out early on as it doesn't show up in the response.env either.
```
  response:
    status:
      code: 403
      message: Forbidden
    headers:
      Content-Type:
      - application/json
```

**Comment by Zorbash** [comment]
With `response.status` you can get the Integer HTTP status code and you can use `response.body` to get the body of the response which may contain useful info about 4xx/5xx errors. Since the HTTP status text is always known beforehand (eg 403 => Forbidden, 418 => I'm a teapot, ..) why would you need it to be accessible from the response object?

**Comment by sbleon** [comment]
@BaneOfSerenity this is implemented in #547, which I hope to have merged soon.

**Comment by sbleon** [comment]
I'm closing this issue in favor of PR #547. Please follow that if you're interested in the fix.
